app/helpers/discord_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DiscordHelper
+  def send_discord(integration, message)
+    Central::Support::Discord.send(real_private_uri(integration.data['private_uri']),
```

Comment:
You need to change here as well, to point to the discord service class in the project instead of support gem